main.go

```diff
@@ -295,6 +309,37 @@ func wrapMain() error {
 		}
 	}
 
+	if vargs.KubeInject {
+		// Inject istio sidecar proxy into pods
+		for _, manifestPath := range pathArg {
+			manifestBase := filepath.Base(manifestPath)
+			manifestDir := filepath.Dir(manifestPath)
+			manifestExt := filepath.Ext(manifestPath)
+			manifestBaseWithoutExt := strings.Replace(manifestBase, manifestExt, "", 1)
+			// output path would resemble "/tmp/.kube.istio.yml"
+			istioOutputPath := filepath.Join(manifestDir, fmt.Sprintf("%s.istio%s", manifestBaseWithoutExt, manifestExt))
+			// use `--includeIPRanges`
+			if len(vargs.IncludeIPRanges) > 0 {
+				err = runner.Run(vargs.IstioctlCmd, "kube-inject", "--istioNamespace", vargs.IstioNamespace, "--includeIPRanges", vargs.IncludeIPRanges, "--filename", manifestPath, "--output", istioOutputPath)
```

Comment:
can you define the command once, then conditionally add to it, to avoid duplication?

Reply:
yeah, wasn't sure how execute a func in go with variadic arguments.. will track that down.